test/controllers/SeriesDetailsControllerSpec.scala

```diff
@@ -69,7 +69,7 @@ class SeriesDetailsControllerSpec extends FrontEndTestHelper {
       seriesDetailsPageAsString must include ("""<option value="" selected>
```

Comment:
Intellij is complaining about some missing type annotations at lines 31 to 33, if you could add the types in for the `val`s. Thanks